### PR TITLE
Constraint all transitive akka-* dependencies

### PR DESCRIPTION
**Pull request by style95**
This may fix the build issue with the Akka family.

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -35,9 +35,14 @@ gradle.ext.openwhisk = [
 
 gradle.ext.scala = [
     version: '2.12.7',
```
Comment (dgrove-oss) [inline settings.gradle:37]: The core repo has `version: 2.12.10`. Should we align the downstream with that as well?

**Comment by rabbah** [comment]
@style95 i did it differently for the go runtime - see https://github.com/apache/openwhisk/issues/5132. If we can afford to wait a bit we can reduce this set to just two.

**Comment by style95** [comment]
@rabbah Updated in the same way.

**Review by rabbah — APPROVED** [review]
Will merge and we can clean up later since upstream may take a bit longer.

**Pull request merged**